eng/internal/NuGet.Template.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json" value="https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json" />
+    <add key="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
+    <add key="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="FeedFromProperties" value="##feed##" />
+  </packageSources>
+  <packageSourceCredentials>
+    <FeedFromProperties>
+      <add key="Username" value="##username##" />
+      <add key="ClearTextPassword" value="##pat##" />
```

Comment:
Are we OK that this is no longer encrypted?

Reply:
The value in hiding this is if someone can get the file but doesn't generally have access to the machine to decrypt it using the build account, it would stop that from leaking secrets.

That seems unlikely: the machines available for public CI (anyone can run anything on these machines by submitting a PR) should *never* be available for internal CI. Even if that weren't the case, since this kind of encryption is account-based, encrypting this PAT wouldn't accomplish much anyway.

There are higher value files to go after if we want to be secure against that kind of thing. However, this change would just add to the pile if we do want to go after that at some point.